src/live_restore/live_restore_fs.c

```diff
@@ -11,6 +11,11 @@
 
 /* This is where basename comes from. */
 #include <libgen.h>
+#include <sys/statvfs.h>
```

Comment:
I'm still waiting on the macOS compile task to run in evergreen due to a backlog.
I'll confirm this compiles on mac/make it macOS compatiable tomorrow morning pending the patch result.